onset_maproom/config-mali-meteo.yaml

```diff
@@ -33,19 +33,19 @@ rr_mrg_chunks:
     X: 24
 rr_mrg_zarr_path: /data/remic/mydatafiles/MaliMeteo/ENACTS/ALL/mrg_precip_daily/
 
-# Ethiopia
-logo: Ethiopia_IRI_98x48.png
-institution: NMA
-zoom: 6
-shapes_adm1: select gid as key, name_1 as label, ST_AsBinary(the_geom) as the_geom
-                  from eth_regions_dd
-name_adm1: Regions
-shapes_adm2: select gid as key, name_2 as label, ST_AsBinary(the_geom) as the_geom
-                  from eth_zones_dd
-name_adm2: Zones
-shapes_adm3: select gid as key, name_3 as label, ST_AsBinary(the_geom) as the_geom
-                  from eth_woredas_dd
-name_adm3: Woredas
+# Mali
+logo: mali.gif
+institution: MaliMétéo
+zoom: 5
```

Comment:
This zoom level feels almost too zoomed in when the user opens the page; I usually would think it is better to have a more zoomed out initial view than a view that does not show all of the data, but maybe that is just a personal opinion. I like it more when the zoom is set to 4

Reply:
Created https://github.com/iridl/python-maprooms/issues/184 "Automatically choose initial zoom level that fits region of interest in window."